projectile/core/models.py

```diff
@@ -113,3 +108,41 @@ class User(AbstractBaseUser, BaseModelWithUID, PermissionsMixin):
     class Meta:
         verbose_name = "System User"
         verbose_name_plural = "System Users"
+
+
+class PasswordReset(NameSlugDescriptionBaseModel):
+    user = models.ForeignKey(
```

Comment:
@FazlulAyanKoushik  for password reset does we require NameSlugDescriptionBaseModel?

Reply:
> @FazlulAyanKoushik for password reset does we require NameSlugDescriptionBaseModel?

In future if we have a feature like, User can request to organization to reset password. then it will require.

Reply:
@FazlulAyanKoushik  here the system is a single vendor , so organization is not necessary